peekingduck/pipeline/nodes/output/screen.py

```diff
@@ -49,19 +53,66 @@ def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
         cv2.namedWindow(self.window_name, cv2.WINDOW_NORMAL)
         cv2.moveWindow(self.window_name, self.window_loc["x"], self.window_loc["y"])
+        self.previous_filename = ""
+        self.previous_width = 0
+        self.previous_height = 0
```

Comment:
These variables are only being assigned to throughout this code and never read from. Are they actually needed?

Reply:
Removed